woocommerce/woocommerce-android
# Bundle editing

Part of: #9541

### Why
As part of the support for extensions milestone 2, we are working on support bundle products in the order creation flow.

### Description
This PR includes the necessary changes to edit a bundled product configuration. Now, on the bottom of the product card, you can select the configuration button and edit the configuration.  To achieve this, I refactored the getConfiguration function into a separate UseCase and added support for variable configurations.


### Testing instructions
1. Open the orders screen
2. Select an order containing a bundled product
3. Tap on edit order
4. Expand on the bundled product
5. Update the order configuration
6. Press on Done
7. Check that the bundled product is updated with the new configuration

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/5e759768-e33f-44d7-aeb1-d57947537ec5
